components/overledger/overledger.app.mjs

```diff
@@ -11,35 +29,55 @@ export default {
     },
   },
   methods: {
-    _baseUrl() {
-      return "https://api.overledger.io";
-    },
     _headers() {
       return {
         "Authorization": `Bearer ${this.$auth.oauth_access_token}`,
         "Content-Type": "application/json",
         "API-Version": "3.0.0",
       };
     },
+    _getBaseUrl() { //conditional for environment url selection.
+      return this.environment === "sandbox"
+        ? "https://api.sandbox.overledger.io"
+        : "https://api.overledger.io";
+    },
     _makeRequest({
-      $ = this, path, ...otherOpts
+      $ = this, baseUrl, path, ...otherOpts
     }) {
       return axios($, {
         ...otherOpts,
-        url: this._baseUrl() + path,
+        url: baseUrl + path,
```

Comment:
Passing the environment selected by the user to _baseUrl method
```suggestion
        url: this._baseUrl(environment)+ path,
```

Reply:
I assume here you meane to replace baseUrl with this._getBaseUrl(environment) and not this._baseUrl(environment).

Reply:
done

Reply:
> I assume here you meant to replace baseUrl with this._getBaseUrl(environment) and not this._baseUrl(environment).

Yes, You're right